crates/ruff_linter/src/rules/pyflakes/snapshots/ruff_linter__rules__pyflakes__tests__F401_F401_24____init__.py.snap

```diff
@@ -0,0 +1,42 @@
+---
+source: crates/ruff_linter/src/rules/pyflakes/mod.rs
+---
+__init__.py:23:8: F401 [*] `sys` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
+   |
+23 | import sys # F401: remove unused
+   |        ^^^ F401
+   |
+   = help: Remove unused import: `sys`
+
+ℹ Unsafe fix
+20 20 | 
+21 21 | 
+22 22 | 
+23    |-import sys # F401: remove unused
+24 23 | 
+25 24 | 
+26 25 | 
+
+__init__.py:43:15: F401 [*] `.unused` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
+   |
+43 | from . import unused # F401: change to redundant alias
+   |               ^^^^^^ F401
+   |
+   = help: Remove unused import: `.unused`
+
+ℹ Safe fix
+40 40 | 
+41 41 | 
+42 42 | 
+43    |-from . import unused # F401: change to redundant alias
+   43 |+from . import unused as unused # F401: change to redundant alias
+44 44 | 
+45 45 | 
+46 46 | 
+
+__init__.py:47:26: F401 `.renamed` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
+   |
+47 | from . import renamed as bees # F401: no fix
+   |                          ^^^^ F401
+   |
+   = help: Remove unused import: `.renamed`
```

Comment:
Can you file an issue to fix this underline and help message? We should probably highlight "renamed as bees"

Reply:
Are you saying that the diagnostic should always apply to the whole alias (name and asname) or are you saying that we should special-case the the renamed-but-unused case?

Reply:
I think that "Remove unused import: `.renamed`" with only "bees" highlighted is very confusing. I'd need to look at more cases to determine what we should change it to — that seems out of scope here.

Reply:
Ok, I'll file an issue to point out that this rule only highlights the as-name part of the alias and that's confusing. :+1:

Reply:
Filed https://github.com/astral-sh/ruff/issues/11223